build_xcframework.sh

```diff
@@ -0,0 +1,134 @@
+#!/bin/bash
+
+# This script will build a universal framework for Xcode.
+# It will contain libraries for iOS (device, simulator) for arm64 and x84_64, as well as a library for macOS for both arm64 and x86_64.
+
+TMP_DIR="./tmp"
+IOS_SIM_ARCHIVE="$TMP_DIR/iOS-Simulator.xcarchive"
+IOS_ARCHIVE="$TMP_DIR/iOS.xcarchive"
+MACOS_ARCHIVE="$TMP_DIR/macOS.xcarchive"
+MAC_CATALYST_ARCHIVE="$TMP_DIR/macCatalyst.xcarchive"
+XCFRAMEWORK="$TMP_DIR/VerovioFramework.xcframework"
+XCFRAMEWORK_ZIP="$XCFRAMEWORK.zip"
+VEROVIO_COMMIT_HASH="$(git rev-parse --short HEAD)"
+VEROVIO_VERSION="$(cat codemeta.json | jq .softwareVersion | sed -e "s/-dev//g" | sed -e "s/\"//g" | sed -e "s/.[0-9]$//g" )"
```

Comment:
Given history of Verovio changes I don't think it is a big thing, but wouldn't last `sed` command be safer with  `"s/\.[0-9]*$//g"` instead of `"s/.[0-9]$//g"`? Current regex expects minor version to never go to double digits and if it does, we might end up with extra dot in version string.

Reply:
Thank you, nice catch. I will fix that.

Reply:
So probably use a `+` instead a `*`? There should be at least one digit I guess.

Reply:
Done